src/peerconnection.cc

```diff
@@ -108,40 +108,40 @@ void PeerConnection::Run(uv_async_t* handle, int status)
     if(PeerConnection::ERROR_EVENT & evt.type)
     {
       PeerConnection::ErrorEvent* data = static_cast<PeerConnection::ErrorEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(NanNew("onerror")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(Nan::New("onerror").ToLocalChecked()));
       v8::Local<v8::Value> argv[1];
-      argv[0] = Exception::Error(NanNew(data->msg));
-      NanMakeCallback(pc, callback, 1, argv);
+      argv[0] = Nan::Error(data->msg.c_str());
+      Nan::MakeCallback(pc, callback, 1, argv);
     } else if(PeerConnection::SDP_EVENT & evt.type)
     {
       PeerConnection::SdpEvent* data = static_cast<PeerConnection::SdpEvent*>(evt.data);
-      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(NanNew("onsuccess")));
+      v8::Local<v8::Function> callback = v8::Local<v8::Function>::Cast(pc->Get(Nan::New("onsuccess").ToLocalChecked()));
       v8::Local<v8::Value> argv[1];
-      argv[0] = NanNew(data->desc);
-      NanMakeCallback(pc, callback, 1, argv);
+      argv[0] = Nan::New(&data->desc);
```

Comment:
`data->desc` is the correct SDP string, but `Nan::New(&data->desc)` is `true`. Trying to figure out how to get this working but v8 and Nan is a new frontier.